### DISCUSSION TITLE
Fortran 77 code not running in VS code

### DISCUSSION THREAD

**Discussion by Mayank140889** [fortran-lang/fortls#410] [Q&A]
Hello,
I am working on one of the old project which was written in old Fortran 77 language syntax fashion with extension filename.for . I want to run the code in VS code and it give me error of "code language is not supported or define". I selected language as Fortran 77 but still code is not running. I also installed modern Fortran extension but the problem is same. kindly Help me to fix the problem @gnikit

**Comment by gnikit** [comment]
Can you post your:
- vscode version
- modern fortran for vscode version
- fortls version
- list the steps you take to get this error (from opening vscode)

**Reply by Mayank140889** [reply]
1. vscode Version - 1.90.0
2. modern Fortran v 3.2.0
3. fortls version 3.1.1
4. when i open Vscode and open my code file named as filename.for. it give me a notification as given below
![Capture4](https://github.com/fortran-lang/fortls/assets/172244342/aba6f97a-bff9-4d88-9ccb-b91bc7a48656)
even after successfully installation of fortls it gave me again one notification
![Capture1](https://github.com/fortran-lang/fortls/assets/172244342/cc6d4b89-818c-4773-b98c-ae1e1fac0060)
